client/flutter/lib/components/page_scaffold/page_scaffold.dart

```diff
@@ -0,0 +1,52 @@
+import './page_header.dart';
+import './share_bar.dart';
+import 'package:flutter/material.dart';
+
+class PageScaffold extends StatelessWidget {
+  final String title;
+  final String subtitle;
+
+  final List<Widget> body;
+
+  final BuildContext context;
```

Comment:
This `context` is not used. The `PageHeader` in the build method used the `context` from its params. Which is working fine.
So this `context` should be removed, as it's also not a good practice to pass contexts to widgets in constructors (they get it through the build method anyway)

Reply:
Done